game-core/src/main/java/games/strategy/triplea/ai/pro/data/ProPurchaseOption.java

```diff
@@ -200,12 +200,11 @@ public double getAmphibEfficiency(final GameData data, final List<Unit> ownedLoc
       final List<Unit> unitsToPlace) {
     final double supportAttackFactor = calculateSupportFactor(ownedLocalUnits, unitsToPlace, data, false);
     final double supportDefenseFactor = calculateSupportFactor(ownedLocalUnits, unitsToPlace, data, true);
-    final double hitPointPerUnitFactor = (3 + hitPoints / quantity);
+    final double hitPointPerUnitFactor = (3 + (double) hitPoints / quantity);
     final double transportCostFactor = Math.pow(1.0 / transportCost, .2);
-    final double hitPointValue = 2 * hitPoints;
     final double attackValue = (amphibAttack + supportAttackFactor * quantity) * 6 / data.getDiceSides();
     final double defenseValue = (defense + supportDefenseFactor * quantity) * 6 / data.getDiceSides();
-    return Math.pow((hitPointValue + attackValue + defenseValue) * hitPointPerUnitFactor * transportCostFactor / cost,
+    return Math.pow(((2 * hitPoints) + attackValue + defenseValue) * hitPointPerUnitFactor * transportCostFactor / cost,
         30) / quantity;
```

Comment:
I didn't notice this during the review of #4099, but even a small change in the first argument to `pow()` here could have drastic changes on the output because it is being raised to the **30th** power.  For example, a 0.1% change in the input will have a 3% change in the output.

Reply:
This is fine.